plane/source/docs/thermal-mode.rst

```diff
@@ -0,0 +1,16 @@
+.. _thermal-mode:
+
+============
+THERMAL Mode
+============
+
+If SOARING is enabled (:ref:`SOAR_ENABLE<SOAR_ENABLE>` = 1 and it has not been disabled by an RC switch (``RCx_OPTION =88``), this mode will idle the throttle, and begin circling searching for vertical lift.
+Navigation is controlled in the same manner as the previous mode, ie if a pilot controlled mode, it remains so. If an auto-navigating mode, it continues the course demanded by that mode. 
```

Comment:
That's not correct, navigation is always towards centre of lift while in THERMAL mode.

Reply:
sorry, written before I understood the mode....thinking it would do the altitude management like SOARING....forgot to remove once I understood it...